commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -157,21 +167,99 @@ cpdef run( filename_trk, path_out, filename_peaks = None, filename_mask = None,
 
     # fiber-tracts from .trk
     print( '\t\t* tractogram' )
-    try :
-        _, trk_hdr = nibabel.trackvis.read( filename_trk )
+    
+    if (path_out is None):
+        raise IOError( 'Path out not defined' )
```

Comment:
Shall we use `IOError` or `RuntimeError` as before? I don't know what the best strategy is

Reply:
I think `RuntimeError`.